webapp/src/components/modals/channel_settings/__snapshots__/edit_channel_settings.test.tsx.snap

```diff
@@ -186,10 +186,38 @@ exports[`components/EditChannelSettings SERVER - should match snapshot after fet
               "label": "Issue Updated: Custom - Affects Version/s",
               "value": "event_updated_Affects Version/s",
             },
+            Object {
+              "label": "Issue Updated: Custom - Component/s",
+              "value": "event_updated_Component/s",
+            },
+            Object {
+              "label": "Issue Updated: Custom - Environment",
+              "value": "event_updated_Environment",
+            },
             Object {
               "label": "Issue Updated: Custom - Epic Link",
               "value": "event_updated_Epic Link",
             },
+            Object {
+              "label": "Issue Updated: Custom - Epic Name",
+              "value": "event_updated_Epic Name",
+            },
+            Object {
+              "label": "Issue Updated: Custom - Issue Type",
+              "value": "event_updated_Issue Type",
+            },
+            Object {
+              "label": "Issue Updated: Custom - Linked Issues",
+              "value": "event_updated_Linked Issues",
+            },
+            Object {
+              "label": "Issue Updated: Custom - Project",
```

Comment:
Need to investigate why Project and other fields around this diff are showing up as available events

Reply:
This is only happening for Jira server, so creating a ticket as it is not blocking for community. https://mattermost.atlassian.net/browse/MM-19810